apiserver/params/params.go

```diff
@@ -414,13 +414,18 @@ type DestroyUnitParams struct {
 
 	// Force controls whether or not the destruction of an application
 	// will be forced, i.e. ignore operational errors.
-	Force bool
+	Force bool `json:"force"`
 
 	// Errors contains errors encountered while applying this operation.
 	// Generally, these are non-fatal errors that have been encountered
 	// during, say, force. They may not have prevented the operation from being
 	// aborted but the user might still want to know about them.
-	Errors []error
+	Errors []error `json:"errors,omitempty"`
```

Comment:
I am pretty sure this needs to be of type []params.Error or else serialisation across the wire will fail spectacularly.